src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -31,15 +31,23 @@ export class GraphStyles {
         css: {
           // color: PfColors.Black,
           content: (ele: any) => {
+            const name = ele.data('service') || ele.data('id');
+            const namespace = name.split('.')[1];
+            const service = name.split('.')[0];
```

Comment:
you should now be able use ele.data('serviceName') and ele.data('namespace') and avoid the split completely.